helm-charts/splunk-otel-collector/values.yaml

```
@@ -1180,6 +1180,12 @@ operator:
       nodejs:
         repository: ghcr.io/signalfx/splunk-otel-js/splunk-otel-js
         tag: v2.6.1
+      dotnet:
+        repository: ghcr.io/signalfx/splunk-otel-dotnet/splunk-otel-dotnet
+        tag: 1.2.1
```

https://github.com/signalfx/splunk-otel-dotnet/pkgs/container/splunk-otel-dotnet%2Fsplunk-otel-dotnet
```suggestion
        tag: v1.2.1
```